### ISSUE TITLE
Limiting total number of requests

### ISSUE THREAD

**Issue by rsiatka** [tsenart/vegeta#684]
#### Question
Hi,
I can see that for **vegeta** the main condition to stop the simulation is time (-_duration_ param). But is there an option to stop the simulation after reaching a certain number of sent requests? 
For example, tools such as: **ab**, **oha**, **hey** in addition to duration, have a parameter -_n_ that is limiting number of requests to run - when this value is reached the simulation ends.

**Comment by noboruma** [comment]
You're welcome @rsiatka 
You can only assess after trying what was the highest rate achievable.
There are many factors that will impact your network connectivity (your NIC, the state of the network, the distance, and so on). So even if you ask for 1M RPS, the tool might not be able to provide you the requested rate depending on physical constraints.

To limit the unknowns, try to test on a private network and always record how many requests were successful.

**Comment by rsiatka** [comment]
Yes @noboruma, totally agree.  In order to simplify the tests and reduce the factors causing uncertainty, I have disabled the NIC and testing my service on **localhost** first, actually on **127.0.0.1** because vegeta, in the absence of a card, cannot resolve localhost on winows. Getting `Get http://localhost:999: lookup localhost on [::1]:53: read udp [::1]:50813->[::1]:53: i/o timeout`
